src/content/docs/apm/agents/php-agent/other-integrations/magento-business-insights.mdx

```diff
@@ -0,0 +1,266 @@
+        
+        If you need to install the agent in a different way, see [PHP agent installation overview](/docs/apm/agents/php-agent/installation/php-agent-installation-overview/).
+
+        </Step>
+    
+    <Step>
+        ## Install the MySQL integration
+
+        Our MySQL integration monitors your databases and collects metrics like database utilization and connections such as slowest queries and active connections by node.
+
+        To install the integration:
+
+            1. Go to **[one.newrelic.com](https://one.newrelic.com/) > Add data**.
+            2. Search for and click **MySQL**.
+            3. Select an account and follow the guided install steps.
+
+If you need to install the agent in a different way, see [MySQL monitoring integration](/install/mysql/).
+
+        </Step>
+    
+    <Step>
+        ## Install the Apache integration
+
+        Our Apache integrations collects network metrics like TCP connections, DNS lookup, HTTPS, SSL, and server and worker statuses.
+
+        To install the integration, follow the steps in our [guided install](https://one.newrelic.com/launcher/nr1-core.explorer?pane=eyJuZXJkbGV0SWQiOiJucjEtY29yZS5saXN0aW5nIn0=&cards[0]=eyJuZXJkbGV0SWQiOiJucjEtaW5zdGFsbC1uZXdyZWxpYy5ucjEtaW5zdGFsbC1uZXdyZWxpYyIsInBhdGgiOiJvaGkiLCJyZWNpcGVOYW1lIjoiYXBhY2hlLW9wZW4tc291cmNlLWludGVncmF0aW9uIiwiYWN0aXZlQ29tcG9uZW50IjoiVlRTT0NvbW1hbmQiLCJhY3RpdmVFbnZpcm9ubWVudCI6ImFwYWNoZS1vcGVuLXNvdXJjZS1pbnRlZ3JhdGlvbiJ9).
+        
+If you need to install the agent in a different way, see [Apache monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/apache-monitoring-integration).
+
+        </Step>
+    
+
+    <Step>
+        ## Install the Elasticsearch integration
+        
+        Our Elasticsearch integration collects metrics like cluster status and current shards. If you're using [Magento version 2.4 and higher](https://magentip.com/install-magento-2-4-x-on-ubuntu-with-elasticsearch/), you'll to install an ElasticSearch application and agent to monitor that application. 
+        
+        To install the integration:
+
+        1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add data**.
+        2. Search for and click on **Elasticsearch**.
+        3. Select an account and follow the guided install steps.
+
+If you need to install the agent in a different way, see [Apache monitoring integration](/docs/infrastructure/host-integrations/host-integrations-list/elasticsearch/elasticsearch-integration).
+
+        </Step>
+    
+
+    <Step>
+        ## Create synthetic monitors
+
+        Our synthetic monitoring feature uses synthetic monitors to test your app in different geolocations and capture server downtime.
```

Comment:
@rahulbasu-nr Does synthetic monitor data populate one of the charts on the business insights dashboard?